File: cpp/day_3_Two City Scheduling.cpp
class Solution {
public:

    int twoCitySchedCost(vector<vector<int>>& costs) {
        int i, ans=0, n=costs.size();
        sort(costs.begin(), costs.end(), [&](vector<int> &c1, vector<int> &c2){
            return (c1[0] - c1[1]) < (c2[0] - c2[1]);
        });
        for(i = 0; i < n/2; i++){
            ans += costs[i][0] + costs[i+n/2][1];
        }
        return ans;
    }
};

File: python3/june/day_3_Two City Scheduling.py
class Solution:
    def twoCitySchedCost(self, costs: List[List[int]]) -> int:
        costs.sort(key=lambda x:x[0]-x[1])
        ans = 0
        n = len(costs)
        for i in range(0, n//2):
            ans += costs[i][0] + costs[i+n//2][1]
        return ans
